docs/user/security/tutorials/how-to-secure-access-to-kibana.asciidoc

```diff
@@ -0,0 +1,133 @@
+
+[float]
+==== Create a space
+
+Create a `Marketing` space for your marketing analysts to use.
+
+. Open the main menu, and select **Stack Management**.
+. Under **{kib}**, select **Spaces**
+. Click **Create a space**.
+. Give this space a unique name. For example: `Marketing`.
+. Click **Create space**.
+
+If you’ve followed the example above, you should end up with a space that looks like this:
+
+[role="screenshot"]
+image::user/security/images/tutorial-secure-access-example-1-space.png[Create space UI]
+
+
+[float]
+==== Create a role
+
+Create a role that describes the privileges you want to grant.
+
+To effectively use dashboards, the marketing analyst will need:
+
+* Access to **read** the data that powers the dashboards
+* Access to **read** the dashboards within the `Marketing` space.
+
+. Open the main menu, and select **Stack Management**.
+. Under **Security**, select **Roles**.
+. Click **Create role**.
+. Give this role a unique name. For example: `marketing_dashboards_role`.
+. For this example, you want to store all marketing data in the `acme-marketing-*` set of indices. To grant this access, locate the **Index privileges** section and enter:
+.. `acme-marketing-*` in the **Index** field.
+.. `read` and `view_index_metadata` in the **Privileges** field.
++
+TIP: You can add multiple patterns of indices, and grant different access levels to each. Click **Add index privilege** to grant additional access.
+. To grant access to dashboards in the `Marketing` space, locate the {kib} section, and click **Add {kib} privilege**:
+.. From the **Spaces** dropdown, select the `Marketing` space.
+.. Expand the **Analytics** section, and select the **Read** privilege for the **Dashboard** feature.
+.. Click **Add Kibana privilege**.
+. Click **Create role**.
+
+If you’ve followed the example above, you should end up with a role that looks like this:
+
+[role="screenshot"]
+image::user/security/images/tutorial-secure-access-example-1-role.png[Create role UI]
+
+
+[float]
+==== Create a user
+
+Now that you created a role, create a user account.
+
+. Navigate to *Stack Management*, and under *Security*, select *Users*.
+. Click *Create user*.
+. Give this user a descriptive username, and choose a secure password.
+. Assign the *marketing_dashboards_role* that you previously created to this new user.
+. Click *Create user*.
+
+[role="screenshot"]
+image::user/security/images/tutorial-secure-access-example-1-user.png[Create user UI]
+
+[float]
+==== Verify
+
+Now that you have a user and role, verify that everything is working correctly.
+
+. Logout of {kib} if you are already logged in.
+. In the login screen, enter the username and password for the account you created.
+
+You’re taken into the `Marketing` space, and the main navigation shows only the *Dashboard* application.
```

Comment:
If you include a "+" on lines 117 and 119, the sentence and image will indent under step 2.

Reply:
Oh that's great! Thanks for the tip